Src/LogData.c

```diff
@@ -16,6 +16,10 @@ static int FAST_LOG_DATA_PERIOD = 50;
 static FATFS fatfs;
 static FIL file;
 
+static int containsHeader = 0;
```

Comment:
We can use a uin8_t for this variable instead of int because this data will always be 1 byte or smaller and will be unsigned. This variable can be a static variable placed above the for loop instead of being a global variable in this file. Static variables in functions retain their value over all calls to that function.